### ISSUE TITLE
DynamoDB Global table V2 - Creation fails with multiple replicas

### ISSUE THREAD

**Issue by giacomomagini** [terraform-providers/terraform-provider-aws#13132]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

0.12.24

### Affected Resource(s)

* aws_dynamodb_table

### Terraform Configuration Files

```hcl
provider "aws" {
  region = "eu-west-1"
}

resource "aws_dynamodb_table" "t" {

  name     = "test-table"
  hash_key = "key"

  stream_enabled   = true
  stream_view_type = "NEW_AND_OLD_IMAGES"

  billing_mode = "PAY_PER_REQUEST"

  attribute {
    name = "key"
    type = "S"
  }

  replica {
    region_name = "us-east-1"
  }

  replica {
    region_name = "us-east-2"
  }

}

```

### Debug Output

https://gist.github.com/giacomomagini/4e308947a2ad7c857ecb2edc3961dbaa

### Expected Behavior

Create a global table with the two replicas `us-east-1` and `us-east-2`.

### Actual Behavior

Error: error creating DynamoDB Table (test-table) replicas: error creating DynamoDB Table (test-table) replicas: ValidationException: Update table operation with more than one create or delete replica actions not allowed

### Steps to Reproduce

1. `terraform init`
2. `terraform plan -out=plan.out <module_path>`
3. `terraform apply plan.out`

### Important Factoids

None.

### References

The first implementation of the code is in this PR (https://github.com/terraform-providers/terraform-provider-aws/pull/12342) but a refactor has been done afterwards in this commit (https://github.com/terraform-providers/terraform-provider-aws/commit/715b45338bdcf825e09ae44e5e5fd9503d3ad1e3)

The issue is that tf sends a single update with all the `replica` create commands while it should send these every create command in a different call.

* #0000

**Comment by joshmyers** [comment]
I wonder if this worked in the original implementation but broke in https://github.com/terraform-providers/terraform-provider-aws/commit/715b45338bdcf825e09ae44e5e5fd9503d3ad1e3#diff-c8928c2dd5459e42d28428384a191ba8L633 @cmonty / @bflad 

There aren't currently any tests to cover multiple replicas.

**Comment by cmonty** [comment]
We did test creating replicas with the original implementation and that worked. I don't see any specific documentation from AWS on this topic.

**Comment by jsalcedo1987** [comment]
I'm facing the same issue with Terraform and Dynamo Global Tables.
Is there any workaround available?

**Comment by giacomomagini** [comment]
@jsalcedo1987 the workaround is adding the replicas one by one. Add, plan, apply for every single replica.

**Comment by giacomomagini** [comment]
@cmonty the documentation is not clear at all but if you look at the tutorial they tell you to run a command for every replica you want to create.
![image](https://user-images.githubusercontent.com/16667545/82532022-bf6c9d00-9b40-11ea-93d0-aabb51515553.png)
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/V2globaltables.tutorial.html

**Comment by bflad** [comment]
Sorry for the hassle here folks 😓 that followup optimization attempt should have been proven via additional acceptance testing. To that end, have submitted the following to add the testing and fix this issue: #13523

**Comment by bflad** [comment]
The fix for this has been merged, verified, and will release with version 2.65.0 of the Terraform AWS Provider, likely tomorrow.
